vsts/pipelines/templates/_releaseBaseImagesStepTemplate.yml

```diff
@@ -27,20 +27,38 @@ steps:
     TargetFolder: '$(Build.ArtifactStagingDirectory)'
 
 - task: ms-devlabs.utilitytasks.task-Shellpp.Shell++@0
-  displayName: 'Pull and create release tags'
+  displayName: 'Pull and create release tags for stretch images'
   inputs:
     type: FilePath
     scriptPath: ./vsts/scripts/tagBaseImagesForRelease.sh
     args: ${{ parameters.baseImageName }}
 
+- task: ms-devlabs.utilitytasks.task-Shellpp.Shell++@0
```

Comment:
this is not required? since you have a separate line already for each flavor?
```
retagImageWithStagingRepository node-runtimeimage-bases-buster.txt $imageName buster
retagImageWithStagingRepository node-runtimeimage-bases-stretch.txt $imageName stretch
```